scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,805 @@
+#! /usr/bin/env python
+"""
+usage: sonic_sku_create.py [-h] [-v] [-f FILE] [-m [MINIGRAPH_FILE]] [-b BASE]
+                           [-r] [-k HWSKU]
+                           [-p] [-vv]
+Create a new SKU
+
+optional arguments:
+  -h, --help            Show this help message and exit
+  -v, --version         show program's version number and exit
+  -f FILE, --file FILE  SKU definition from xml file. -f OR -m must be provided when creating a new SKU
+  -m [MINIGRAPH_FILE], --minigraph_file [MINIGRAPH_FILE]
+                        SKU definition from minigraph file. -f OR -m must be provided when creating a new SKU
+  -b BASE, --base BASE  SKU base definition
+  -r, --remove          Remove SKU folder
+  -k HWSKU, --hwsku HWSKU
+                        SKU name to be used when creating a new SKU or for  L2 configuration mode
+  -p, --print           Print port_config.ini without creating a new SKU
+  -vv, --verbose        Verbose output
+
+
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import re
+import subprocess
+import traceback
+import sys
+import shutil
+import copy
+from collections import OrderedDict
+
+from tabulate import tabulate
+from lxml import etree as ET
+from lxml.etree import QName
+
+minigraph_ns = "Microsoft.Search.Autopilot.Evolution"
+minigraph_ns1 = "http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"
+INTERFACE_KEY = "Ethernet"
+
+test_envir = None
```

Comment:
Could you remove test related code out of this script?

Reply:
Yes, I will remove test related code of the script. So, I may have to remove the pytest for testing the SKU creator tool also. Is that okay?